package.json

```
@@ -7,8 +7,8 @@
         "start": "node index.js",
```

Also needs the flag here
```suggestion
        "start": "node --max-http-header-size 80000 index.js",
```

---

Thanks, I have now done this.